include/odp/api/spec/ipsec.h

```diff
@@ -762,6 +762,37 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * Retrieve IPSEC SA information
+ */
+typedef struct odp_ipsec_sa_info_t {
+	/** IPSEC Security Association (SA) parameters */
+	odp_ipsec_sa_param_t param;
+
+	/** IPSEC SA direction dependent internal parameters */
+	union {
+		/** Inbound specific internal parameters */
+		struct {
+			/** Antireplay window top
+			 *
+			 * Sequence number representing the top of the
+			 * antireplay window.
+			 */
+			uint64_t ar_top;
+		} inb;
```

Comment:
Similarly to the union in odp_ipsec_sa_param_t, I'd prefer to use full words as struct named for "inbound" and "outbound" (inb->inbound).

Reply:
Also ar_top should be extended to "antireplay_top" or something similar. We use full word "antireplay" in other parts of the API.

Reply:
> 
> 
> Similarly to the union in odp_ipsec_sa_param_t, I'd prefer to use full words as struct named for "inbound" and "outbound" (inb->inbound).

Agreed. Will update it to inbound and outbound in the next revision.


> 
> 
> Also ar_top should be extended to "antireplay_top" or something similar. We use full word "antireplay" in other parts of the API.

Would "antireplay_window_top" be fine?

Reply:
In V2, updated the following:
1. struct names in sa_info now uses full words: inbound and outbound.
2. Renamed ar_top to antireplay_window_top.